src/components/IconButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { styled } from '../styles';
+import { H3 } from "./Text";
+import { MdSmokeFree } from 'react-icons/md';
+
+const IButtonWrapper = styled.div`
+  width: max-content;
+  height: max-content;
+  padding-top: 8px;
+  padding-left: 32px;
+  border-radius: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+`;
+
+type Props = {
```

Comment:
props convention should be:
1. capitalize I in the beginning
2. component name
3. 'props' in the end